lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -245,68 +320,103 @@ def add_merchant_description(post, options)
       end
 
       def url(action)
-        "#{(test? ? test_url : live_url)}/pts/v2/#{action}"
+        case action
+        when /customers/
+          "#{(test? ? test_url : live_url)}/tms/v2/#{action}"
+        when 'instrumentidentifiers', 'paymentinstruments'
+          "#{(test? ? test_url : live_url)}/tms/v1/#{action}"
```

Comment:
❓ Question:

Is this a typo or you actually want to use the `v1` API version here ?